.semaphore/semaphore.yml

```diff
@@ -43,6 +43,12 @@ global_job_config:
       - echo $SEMAPHORE_WORKFLOW_ID
       - echo $SEMAPHORE_GIT_REPO_SLUG
       - checkout
+      - |
+        if [[ "${SEMAPHORE_ORGANIZATION_URL}" != *".semaphoreci.com" ]]; then
+          . vault-setup
+          . vault-sem-get-secret ci-reporting
+          . vault-sem-get-secret sox-semaphore-build-info
```

Comment:
Are you sure you are using these secrets? If so, this is fine, but at the least, I don't think this is a SOX repo, so I don't think the `sox-semaphore-build-info` is needed. It looks like you are using Testbreak below with `emit-ci-metrics -p -a test-results`. That consumes from `ci-reporting`.

Reply:
let me check

Reply:
yeah, I just tested it without it and it's still able to get the secret